Add link to PolyScope X version to README

urfeex opened the pull request:
Add a note to the README, that there is a separate version for PolyScope X.

urfeex committed:
Add link to PolyScope X version to README

urrsk commented on README.md:
```diff
@@ -1,4 +1,10 @@
+> version, please see [Universal_Robots_ToolComm_Forwarder_URCapX](https://github.com/UniversalRobots/Universal_Robots_ToolComm_Forwarder_URCapX).
+
+
 Package to support relaying the RS-485 communication device in a UR robot to a remote PC. This feature is only supported on the e-series.
```
```suggestion
Package to support relaying the RS-485 communication device in a UR robot to a remote PC. This feature is supported on the e-series and the UR series.
```

urfeex committed:
Update notes of series

urrsk commented on README.md:
```diff
@@ -23,7 +29,7 @@ socat pty,link=${LOCAL_DEVICE_NAME},raw,ignoreeof,waitslave tcp:${ROBOT_IP}:5432
 
 ## Use inside a ROS application
-The [`ur_robot_driver`](http://wiki.ros.org/ur_robot_driver) has a [convenience script](https://github.com/UniversalRobots/Universal_Robots_ROS_Driver/blob/master/ur_robot_driver/scripts/tool_communication) wrapping the above `socat` call. When running the driver, the e-Series launchfiles provide a [flag](https://github.com/UniversalRobots/Universal_Robots_ROS_Driver/blob/master/ur_robot_driver/l ...
+The [`ur_robot_driver`](http://wiki.ros.org/ur_robot_driver) has a [convenience script](https://github.com/UniversalRobots/Universal_Robots_ROS_Driver/blob/master/ur_robot_driver/scripts/tool_communication) wrapping the above `socat` call. When running the driver, the e-Series and UR series launchfiles provide a [flag](https://github.com/UniversalRobots/Universal_Robots_ROS_Driver/blob/master/ur_ ...
```
```suggestion
The [`ur_robot_driver`](http://wiki.ros.org/ur_robot_driver) has a [convenience script](https://github.com/UniversalRobots/Universal_Robots_ROS2_Driver/blob/main/ur_robot_driver/scripts/tool_communication.py) wrapping the above `socat` call. When running the driver, the e-Series and UR series launchfiles provide a [flag](https://github.com/UniversalRobots/Universal_Robots_ROS2_Driver/blob/236cb736f92f212c75eefdb20e08f7bb38366e15/ur_robot_driver/launch/ur_rsp.launch.py#L62) for automatically activating the tool communication. The launchfiles also allow setting up communication parameters on the robot directly.
```

urfeex replied:
I think if we swap to the ROS 2 driver here, we should also not link the ROS Wiki page. Maybe linking to the package on index.ros.org might be an option?

urfeex replied:
@urrsk any input o that?

urrsk replied:
> [@urrsk](https://github.com/urrsk) any input o that?
Yes, lets link to our own documentation or a place where the likelihood of url changes is minimal

urfeex replied:
I've updated the PR accordingly.

urfeex committed:
Update linkts to ROS 2 driver

Since this can be used with ROS 2, as well and ROS (1) is EOL, the
README should point to the ROS 2 driver.
